eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/ExponentialRetry.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.eventhubs;
+
+import java.time.Duration;
+
+/**
+ * A policy to govern retrying of messaging operations in which the delay between retries
+ * will grow in an exponential manner, allowing more time to recover as the number of retries increases.
+ */
+public final class ExponentialRetry extends Retry {
+
+    public static final Duration TIMER_TOLERANCE = Duration.ofSeconds(1);
```

Comment:
nit: add a blank line between `public static` and `private final` variables. It makes it easier to visually group the different types of variables.